JSON.cpp

```diff
@@ -1,10 +1,22 @@
 #include "JSON.h"
+#include <iostream>
```

Comment:
Csupán egy megjegyzés, ha jól látom az iostream include már nem kell ide, de aztán könnyen lehet az is, hogy félrenéztem.

Reply:
Upsz.

Reply:
Oh fene, pedig kitöröltük szegénykét, mégis bent maradt :(